### ISSUE TITLE
Track button usage

### ISSUE THREAD

**Issue by repsej2003** [bitfocus/companion#2900]
### Is this a feature relevant to companion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

There should be a counter on each button, that tells how many times the button has been activated since the counter has been reset.

### Usecases

This is a way to track which buttons is use many times and which are barely used. This is helpful for places where the person who are using it are different than the people who set it up companion. And these feature would help them optimize the layout.

Plus it is fun to see how many times a button have been used over time, if you use it daily.

**Comment by thedist** [comment]
Companion logs show button press/releases, so one option could be to use the logs to get the button press data you need.

**Comment by dnmeid** [comment]
Personally I don't think that this is a feature which many users are missing or would need. Also we would need to find a way to implement this in the UI without wasting a lot of space for such a very occasional needed information. And last there are existing ways to do counters on demand.
But anyhow we value your feedback and ideas for features and so I think maybe we leave this open for a while and see if it evolves into something.

**Comment by repsej2003** [comment]
Yeah i would agree there is more important features to add, i just had the idea, and would like to put it out there.

Thank you for you great work
